### PR TITLE
[Docbug][Implicitgrant][3874]

### PR DESCRIPTION
**Pull request by v-rodriguezd**
Added note and removed statement of Enable implicit grant by checking the following boxes: ✔ ID Token ✔ Access Token. Please review.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/auth-aad-sso.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6e7c3bccb85c01e6742ef6774fc808ab6efbd858/msteams-platform/tabs/how-to/authentication/auth-aad-sso.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1a1e13e3-e803-4723-9c93-0eb7d1cdfe75) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/auth-aad-sso-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/64f93b7114d7b307f3a88b31529da4a64d0ea811/msteams-platform/bots/how-to/authentication/auth-aad-sso-bots.md) | 86 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cdee4322-c47a-457e-92c5-c97d589cded5) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/authentication/auth-aad-sso.md](https://github.com/MicrosoftDocs/msteams-docs/blob/64f93b7114d7b307f3a88b31529da4a64d0ea811/msteams-platform/tabs/how-to/authentication/auth-aad-sso.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3b883ceb-066d-484d-affe-86a4c1c615d5) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — APPROVED** [review]

**Review by v-abirade — DISMISSED** [review]
Approved

**Comment by v-abirade** [comment]
Hello @aosolis,

Request you to review the change.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/auth-aad-sso.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3ed33dbe286bd85ad7dfa50caa3cc5e9cbf5e05e/msteams-platform/tabs/how-to/authentication/auth-aad-sso.md) | 96 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a6272708-f611-40d4-a683-b8096a0a19f0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by v-abirade** [comment]
Hi @aosolis,

Please review the changes,

File: msteams-platform/tabs/how-to/authentication/auth-aad-sso.md [hunk 1]
```diff
@@ -101,18 +100,17 @@ This section describes the tasks involved in creating a Teams tab that uses SSO.
 
 1. Navigate to **Authentication**.
 
-    If an app has not been granted IT admin consent, users have to provide consent the first time they use an app.
+    If an app hasn't been granted IT admin consent, users have to provide consent the first time they use an app.
```
Comment (aosolis) [inline msteams-platform/tabs/how-to/authentication/auth-aad-sso.md:103]: This part is very important, but it's buried in a section about configuring the AAD application.

File: msteams-platform/tabs/how-to/authentication/auth-aad-sso.md [hunk 2]
```diff
@@ -101,18 +100,17 @@ This section describes the tasks involved in creating a Teams tab that uses SSO.
...
 
     To enter a redirect URI:
     * Select **Add a platform**.
     * Select **web**.
-    * Enter the **redirect URI** for your app. This is the page where a successful implicit grant flow redirects the user. This is the same fully qualified domain name that you entered in step 5 followed by the API route where an authentication response is sent. If you are following any of the Teams samples, this is `https://subdomain.example.com/auth-end`.
+    * Enter the **redirect URI** for your app. The redirect URI is the page where a successful implicit grant flow redirects the user. This URI is the same fully qualified domain name that you entered in step 5. It's also followed by the API route where an authentication response is sent. If you're following any of the Teams samples, the URI is `https://subdomain.example.com/auth-end`.
```
[the remark this answers is not in the document]
Comment (v-rodriguezd) [reply msteams-platform/tabs/how-to/authentication/auth-aad-sso.md:108]: @Wajeed-msft could you please have a look at this comment and confirm what change needs to be done.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/auth-aad-sso.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1b273ad7792175c44e24a56ad65f970700b8968e/msteams-platform/tabs/how-to/authentication/auth-aad-sso.md) | 96 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/631a0b5f-0d58-4222-8eba-0ea4ce2142a4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by v-abirade** [comment]
Awaiting inputs.

File: msteams-platform/tabs/how-to/authentication/auth-aad-sso.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (AbhijeetThakur-msft) [reply msteams-platform/tabs/how-to/authentication/auth-aad-sso.md]: @Wajeed-msft, @aosolis checked one sample and it works with implicit grant off. The sample covers sending activity notification [sample reference](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/graph-activity-feed/nodejs) with required permissions for it.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/auth-aad-sso.md](https://github.com/MicrosoftDocs/msteams-docs/blob/027b39bf49a4f4c31413ef76d5b776529fa9b763/msteams-platform/tabs/how-to/authentication/auth-aad-sso.md) | 96 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ca4d6c6a-33f3-4a38-b1c0-7e8413e715a6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**